native/src/routes/search/components/SearchModal.js

```diff
@@ -99,6 +100,9 @@ class SearchModal extends React.Component<PropsType, StateType> {
 
   onItemPress = (category: { path: string, ... }) => {
     const { cityCode, language, navigateTo } = this.props
+    const { query } = this.state
+
+    sendTrackingSignal({ signal: { name: SEARCH_FINISHED_SIGNAL_NAME, query, url: category.path } })
```

Comment:
I think `category.path` is not an url, is it?

Reply:
fixed it.